static/css/main.css

```diff
@@ -27,4 +27,10 @@ label{
 
 input{
     margin-bottom:1rem;
+}
+
+.post{
```

Comment:
what is post ?

Reply:
tweet post. I used this for some design

Reply:
tweet post is wrong. 

`post` doesn't exists as a domain object for our project. Get rid of it